### ISSUE TITLE
Error unable to verify the first certificate *** Could not merge components ***

### ISSUE THREAD

**Issue by samuelrc-test** [microsoft/BotFramework-Composer#9665]
Hello, 

I've searched and I've founded a similar issue with other error message, but the proposed solution didn't work for this case, that's why I opened this new issue.

I am working with Bot framework composer version 2.1.2, but when I try to install de CLU recognizer package, it shows me an error "A problem occurred during the install of this Component: Error unable to verify the first certificate *** Could not merge components ***.

And when I try to see the list of installed packages it shows me a similar error "Could not load component list:Error unable to verify the first certificate *** Could not merge components ***"

You can see that both errors point to the same thing "Error unable to verify the first certificate"

I've tried several versions of Bot Composer like 2.0.0 ; 2.0.1 and the latest nightly version [v2.1.3-nightly.364675.1a93f21] but all of them failed.

thanks for your help.
![Screenshot 2023-10-30 084429](https://github.com/microsoft/BotFramework-Composer/assets/57732145/a155a01b-d6ae-4136-aa21-981af3b558e6)

![Screenshot 2023-10-30 084008](https://github.com/microsoft/BotFramework-Composer/assets/57732145/281059ce-625f-4932-bbf8-54de61eef22f)

**Comment by tracyboehrer** [comment]
Does this only happen with CLU Recognizer?

**Comment by samuelrc-test** [comment]
No. It happens with any package, just I need CLU Recognizer to start migration of our chatbots from Luis to CLU.

**Comment by dmvtech** [comment]
Hi @samuelrc-test 

Sorry for the delay. Can you please test again and let us know if you still have this issue?

**Comment by samuelrc-test** [comment]
Hi @dmvtech 
I've tried and It failed again, same error message.

thanks in advance

**Comment by YuZeSun0203** [comment]
I am experiencing the same issue these days

**Comment by ceciliaavila** [comment]
Hi @samuelrc-test,
We couldn't reproduce the issue. In Composer 2.1.2, we are getting the `certificate has expired` error, but it's solved in the nightly version.
We found a few similar issues (certificate-related errors) and they were caused by a firewall blocking Composer's requests (example: [#9546](https://github.com/microsoft/BotFramework-Composer/issues/9546)).
Is this your case? Are you running Composer behind a firewall?
